test/core-flags/context-flag.test.js

```diff
@@ -1,12 +1,26 @@
 'use strict';
 
-const { run } = require('../utils/test-utils');
+const { resolve } = require('path');
+const { run, isWindows } = require('../utils/test-utils');
 
 describe('--context flag', () => {
     it('should allow to set context', () => {
-        const { stderr, stdout } = run(__dirname, ['--context', '/test-context-path']);
+        const { stderr, stdout, exitCode } = run(__dirname, ['--context', './']);
 
         expect(stderr).toBeFalsy();
-        expect(stdout).toContain('test-context-path');
+        expect(exitCode).toBe(0);
+        if (isWindows) {
+            expect(stdout).toContain('core-flags');
```

Comment:
We can normalize this for windows

Reply:
how?

Reply:
`string`.replace(/\\/, '/') :smile: Just change windows slashes on linux